Source/WebCore/html/CanvasBase.cpp

```diff
@@ -356,6 +365,29 @@ bool CanvasBase::postProcessPixelBufferResults(Ref<PixelBuffer>&& pixelBuffer) c
     return false;
 }
 
+RefPtr<ImageBuffer> CanvasBase::createImageForNoiseInjection() const
+{
+    RefPtr context = canvasBaseScriptExecutionContext();
+    if (!context)
+        return { };
+
+    auto seed = static_cast<unsigned>(context->noiseInjectionHashSalt().value_or(0));
+    auto buffer = ImageBuffer::create(size(), RenderingPurpose::Unspecified, 1, DestinationColorSpace::SRGB(), ImageBufferPixelFormat::BGRA8);
+    if (!buffer)
+        return { };
+
+    auto componentValues = WeakRandom { seed }.getUint32();
+    auto fillColor = SRGBA<uint8_t> {
```

Comment:
I guess it doesn't matter since the fill color is just random, but you do reverse the bits from `componentValues`! 😛

Reply:
Right. I suppose I could reverse the order for clarity? (such that R gets the high byte and A gets the low byte)... Shouldn't make a tangible difference though.